docker-build.sh

```diff
@@ -4,6 +4,16 @@ git submodule update --init --recursive
 
 mkdir -p build
 
+NCPUS=""
+if [ "$LIMIT_CORES" -ge 1 ]; then
+    NCPUS="--cpus $LIMIT_CORES"
+fi
+
+PRECMD="${PRECMD:-true}"
+POSTCMD="${POSTCMD:-true}"
+FLAGS="$(echo "$ADD_FLAGS" | sed -Ee 's|\s+|\n|g')"
+echo -e "${FLAGS}" > build/flags.local.gn
```

Comment:
Do we need to create `flags.local.gn` here if we can just create the file ourselves separately?